src/style.css

```diff
@@ -0,0 +1,87 @@
+
+/* Fonts */
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+/* Styles */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.formField {
+  position: relative;
+  margin-top: 20px;
+}
+
+/* Inputs */
+
+.input {
+  font-family: inherit;
+  font-weight: 800;
```

Comment:
text should become thinner on blur
<img width="237" alt="image" src="https://user-images.githubusercontent.com/66749919/202843235-8836e781-2099-4fb6-817e-c297074fc55f.png">

Reply:
Hi. Sorry, but I'm don't understand what does it mean "text should become thinner on blur". I'm must  change .input or :focus? Can't find any blur styles?